js/views/SocialBtns.js

```diff
@@ -0,0 +1,63 @@
+import app from '../app';
+import loadTemplate from '../utils/loadTemplate';
+import { followedByYou, followUnfollow } from '../utils/follow';
+
+import BaseVw from './baseVw';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    if (!options.targetID) throw new Error('You must provide a targetID');
+
+    super(options);
+    this.options = options;
+
+    this._state = {
+      following: followedByYou(options.targetID),
+      processing: false,
```

Comment:
Could we call this isFollowing instead of the generic processing? The latter wouldn't work well if you had a need to add a seconds processing button in here.